### DISCUSSION TITLE
Get Hidden Dialogs

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#10288] [Q&A] [answered]
Hi,
While we can get Opened Dialog using `DialogOverlayService` , is there any way to get Dialogs that are hidden using `hide()` ?

Thanks

**Comment by vince-fugnitto** [comment, marked as answer]
@maheshrajrp given that most methods in `DialogOverlayService` are protected I assume you are doing the following to get the current dialog?

```ts
const currentDialog = DialogOverlayService.get()['currentDialog']
```

Using the same idea you could do:

```ts
const dialogs = DialogOverlayService.get()['dialogs'];
```

Note: it is probably bad practice to have many opened dialogs overlayed which are hidden.

**Reply by maheshrajrp** [reply]
@vince-fugnitto Thanks. Thanks for the note too. I'm currently using this to create a context to evade opening multiple dialogs using shortcuts.
